src/codegen/generator.py

```diff
@@ -41,9 +52,12 @@ def generate(dest):
 
     codegen_metadata = _get_metadata()
 
-    for info in codegen_metadata["script_info"]["modules"]:
+    for info in codegen_metadata["script_info"]["modulesToGenerate"]:
         _generate_file(codegen_metadata, info["templateFile"], dest / info["relativeOutputPath"])
 
     _generate_file(codegen_metadata["enums"], "error_codes.mako", dest / "error_codes.py")
 
     _generate_file(codegen_metadata, "constants.mako", dest / "constants.py")
+
+    for info in codegen_metadata["script_info"]["modulesToCopy"]:
```

Comment:
the for can be inside the copy method and finding parent directory can be outside of for loop